File: content/learn/arrays-easy/05-left-rotate-by-one/starter.cpp
#include <iostream>
#include <vector>
using namespace std;

void leftRotateByOne(vector<int>& a) {
    // TODO: rotate a left by one position in place.
}

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; i++) cin >> a[i];
    leftRotateByOne(a);
    for (int i = 0; i < n; i++) {
        if (i) cout << " ";
        cout << a[i];
    }
    cout << "\n";
    return 0;
}

File: content/learn/arrays-easy/05-left-rotate-by-one/starter.py
def left_rotate_by_one(a):
    # TODO: rotate a left by one position in place.
    pass


n = int(input())
a = list(map(int, input().split())) if n > 0 else []
left_rotate_by_one(a)
print(" ".join(str(x) for x in a))
